elements/pf-popover/test/pf-popover.spec.ts

```diff
@@ -80,48 +64,49 @@ describe('<pf-popover>', function() {
       expect(document.createElement('pf-popover'))
         .to.be.an.instanceof(PfPopover);
     });
-    it('should not report anything to assistive technology', expectA11ySnapshot());
+    it('should not report anything to assistive technology', () => expectA11ySnapshot());
   });
 
   describe('with a slotted trigger; and with heading, body, and footer attributes', function() {
-    /** Setup the a11y tree snapshot expected results for this suite */
-    const snapshots = {
-      opened: [
-        {
-          name: 'Toggle popover',
-          role: 'button',
-        },
-        {
-          name: 'Close popover',
-          role: 'button',
-        },
-        {
-          name: 'Popover heading',
-          role: 'heading',
-        },
-        {
-          name: 'Popovers are triggered by click rather than hover.',
-          role: 'text',
-        },
-        {
-          name: 'Popover footer',
-          role: 'text',
-        },
-      ],
-      closed: [
-        {
-          name: 'Toggle popover',
-          role: 'button',
-        }
-      ],
-    };
+    // these tests are flaky, soo...
+    beforeEach(resetElement);
+    beforeEach(nextFrame);
+    beforeEach(resetElement);
+    beforeEach(nextFrame);
+    beforeEach(resetElement);
+    beforeEach(nextFrame);
```

Comment:
i found that the tests were flaky. I could reliably get green if i `only`'d the specific test that failed in that block, but when running the entire file it would fail. I think these lines are necessary to unflake the tests.

I'm in favour of removing these and replacing with something more comprehensive... but like... after release

Reply:
also FYI the `element` reference is needed for `updateComplete`

Reply:
> also FYI the `element` reference is needed for `updateComplete`

D'oh, the abstraction of `updateComplete()` to a async function got me... wasn't seeing direct references to `element` on initial scan.  Move along nothing to see here :)